<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="CentralContentWidget">
   <layout class="QHBoxLayout" name="CentralContentWidgetLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="SplitterBody">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="LeftColumnWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="1" column="1">
         <widget class="QScrollArea" name="ThumbnailArea">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="ThumbnailAreaWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>270</width>
             <height>374</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QGridLayout" name="ThumbnailAreaLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinAndMaxSize</enum>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="RightColumnWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>380</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="RightColumnWidgetLayout">
        <item>
         <widget class="QLabel" name="PreviewLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="FileInfoGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="title">
           <string>File info</string>
          </property>
          <layout class="QFormLayout" name="FileInfoGroupBoxLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="TitleFileNameLabel">
             <property name="text">
              <string>File name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="FileNameLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="TitleResolutionLabel">
             <property name="text">
              <string>Resolution</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="ResolutionLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="OpenAction"/>
    <addaction name="RefreshAction"/>
    <addaction name="CloseAction"/>
   </widget>
   <widget class="QMenu" name="SortMenu">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="ByColorAction"/>
    <addaction name="separator"/>
    <addaction name="ShuffleAction"/>
    <addaction name="UnsortedAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="MoreColumnsAction"/>
    <addaction name="LessColumnsAction"/>
    <addaction name="separator"/>
    <addaction name="SlideshowAction"/>
    <addaction name="StopSlideshowAction"/>
    <addaction name="separator"/>
    <addaction name="ToggleFileInfoAction"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="SortMenu"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <action name="OpenAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="CloseAction">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="RefreshAction">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="ByColorAction">
   <property name="text">
    <string>By Color</string>
   </property>
  </action>
  <action name="UnsortedAction">
   <property name="text">
    <string>Unsorted</string>
   </property>
  </action>
  <action name="MoreColumnsAction">
   <property name="text">
    <string>More columns</string>
   </property>
  </action>
  <action name="LessColumnsAction">
   <property name="text">
    <string>Less columns</string>
   </property>
  </action>
  <action name="SlideshowAction">
   <property name="text">
    <string>Slideshow</string>
   </property>
  </action>
  <action name="StopSlideshowAction">
   <property name="text">
    <string>Stop slideshow</string>
   </property>
  </action>
  <action name="ShuffleAction">
   <property name="text">
    <string>Shuffle</string>
   </property>
  </action>
  <action name="ToggleFileInfoAction">
   <property name="text">
    <string>Toggle FileInfo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
